package.json

```diff
@@ -36,26 +36,26 @@
   "dependencies": {
     "bson": "~1.1.4",
```

Comment:
1.1.5

Reply:
hmm I dont think we want to do this cause mongodb and mongo-extended-json are using 1.1.4? if we want to do this I'll have to add a resolution.